config/src/config/mod.rs

```diff
@@ -118,7 +119,7 @@ impl RoleType {
     }
 }
 
-impl std::str::FromStr for RoleType {
+impl FromStr for RoleType {
```

Comment:
Did this improve coverage? or did we need the trait for other purposes?

Reply:
@davidiw, we needed the trait for tests I added for the role type conversion.